apps/full-site-editing/full-site-editing-plugin/premium-content/blocks/logged-out-view/index.js

```diff
@@ -18,9 +18,9 @@ const name = 'premium-content/logged-out-view';
 const category = getCategoryWithFallbacks( 'design', 'common' );
 /**
  * @typedef {object} Attributes
- *
```

Comment:
Not sure where this changes comes from... when I run `yarn`, the file got changed automatically 🤷